KhrystynaVehera/HW7/Task3.py

```diff
@@ -0,0 +1,12 @@
+word_to_count = input("Please write a word to calculate letters: ")
+
+
+def letters_counter(word_to_count):
+    letters_dict = dict()
+
+    for i in set(word_to_count):
```

Comment:
good idea
look at this version
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result